client/my-sites/plans/current-plan/jetpack-checklist/index.tsx

```diff
@@ -172,47 +133,166 @@ class JetpackChecklist extends PureComponent< Props > {
 					progressText={ translate( 'Your Jetpack setup progress' ) }
 				>
 					<Task
-						{ ...JETPACK_CHECKLIST_TASK_PROTECT }
+						id="jetpack_task_protect"
+						title={ translate(
+							"We've automatically protected you from brute force login attacks."
```

Comment:
Present tense here (I changed that in #33666, easy to miss when rebasing)

```suggestion
							"We're automatically protecting you from brute force login attacks."
```